src/main/java/com/ericsson/gerrit/plugins/eiffel/events/generators/EiffelEventGenerator.java

```diff
@@ -0,0 +1,57 @@
+/*
+   Copyright 2019 Ericsson AB.
+   For a full list of individual contributors, please see the commit history.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+package com.ericsson.gerrit.plugins.eiffel.events.generators;
+
+import java.net.InetAddress;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.UnknownHostException;
+
+public class EiffelEventGenerator {
+    protected static final String META_SOURCE_NAME = "Eiffel Gerrit Plugin";
```

Comment:
Shouldn't meta source name be named with the actual source name, e.g. gerrit hostname.

Reply:
No, meta source is just the name of the source which we after discussions some time ago chose as this :)

Reply:
See Eiffel documentation for reference: https://github.com/eiffel-community/eiffel/blob/master/eiffel-vocabulary/EiffelSourceChangeCreatedEvent.md#metasourcename

Reply:
Ok,,, are we using "meta.source.host" somewhere in the event so it can traced where or which specific git host that sent the event,, since in future we can send SCC event from GitLab, which is a different source sender.

Reply:
In the generator class you can see that we set meta.source.host and meta.source.uri

Reply:
Yes we are using it :)

Reply:
ok.